virttest/utils_test/libvirt.py

```diff
@@ -3615,6 +3615,54 @@ def customize_libvirt_config(params,
     return obj_conf
 
 
+def remove_key_in_conf(value, conf_type="libvirtd",
+                       remote_params=None, restart_libvirt=False):
+    """
+    Remove settings in configuration file on local or remote
+    and restart libvirtd if needed
+
+    :param value: The value to delete
```

Comment:
example of value, how to provide will help user.
Given that you are expecting it as a list and modify type even user provides string, it is good to provide the examples.

Reply:
Thanks for your review, updated.